NEWS.md

```diff
@@ -0,0 +1,18 @@
+# New in snapd 2.59:
+
+* Extra kernel command line via snap configuration
```

Comment:
Support setting extra kernel command line parameters via snap configuration and under a gadget allow-list